### ISSUE TITLE
[Accessibility]: Notebooks conflict caused by arrow key navigation 

### ISSUE THREAD

**Issue by marisak** [sourcegraph/sourcegraph#35166]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34197

### Problem description

Search notebooks have a behavior where a notebook can be scrolled by using the up/down arrow keys. This seems to cause confusion with the keyboard navigation, because it also focuses each block. 

### Expected behavior

I'm not sure there's a clear expectation. On one hand, scrolling a page with the up/down arrow keys is expected behavior. On the other hand, focusing the blocks is unexpected.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.

**Comment by limitedmage** [comment]
@almeidapaulooliveira Can you take a look at this issue and decide if any action is required here? Unsure if this is a bug or expected behaviour, and what the fix could be.

**Comment by limitedmage** [comment]
Proposed solution:

1. Only move block focus if modifier key is not pressed (so opt+up/down would not move focus)
2. Set default focus on page to title or `run all blocks` button
3. Disable down arrow focusing first block
